docs/release_notes/release-3.0.md

```diff
@@ -1,5 +1,39 @@
 # StarRocks version 3.0
 
+## 3.0.5
+
+Release date: August 16, 2023
+
+### New Features
+
+- Supports aggregate functions [COVAR_SAMP](../sql-reference/sql-functions/aggregate-functions/covar_samp.md), [COVAR_POP](../sql-reference/sql-functions/aggregate-functions/covar_pop.md), and [CORR](../sql-reference/sql-functions/aggregate-functions/corr.md).
+- Supports the following [window functions](../sql-reference/sql-functions/Window_function.md): COVAR_SAMP, COVAR_POP, CORR, VARIANCE, VAR_SAMP, STD, and STDDEV_SAMP.
+
+## Improvements
+
+- Added more prompts in the error message `xxx too many versions xxx`. [#28397](https://github.com/StarRocks/starrocks/pull/28397)
+- Dynamic partitioning further supports the partitioning unit to be year. [#28386](https://github.com/StarRocks/starrocks/pull/28386)
+- The partitioning field is case-insensitive when expression partitioning is used at table creation and [INSERT OVERWRITE is used to overwrite data in a specific partition](../table_design/expression_partitioning.md#load-data-into-partitions). [#28309](https://github.com/StarRocks/starrocks/pull/28309)
+
+## Bug Fixes
+
+Fixed the following issues:
+
+- Incorrect table-level scan statistics in FE cause inaccurate metrics for table queries and loading. [#27779](https://github.com/StarRocks/starrocks/pull/27779)
+- The query result is not stable if the sort key is modified for a partitioned table. [#27850](https://github.com/StarRocks/starrocks/pull/27850)
+- The version number for a tablet is inconsistent between the BE and FE after data is restored. [#26518](https://github.com/StarRocks/starrocks/pull/26518/files)
+- The number of buckets is automatically inferred as 0 if the number of buckets is not specified at the creation of a Colocation table, which causes failure of adding new partitions. [#27086](https://github.com/StarRocks/starrocks/pull/27086)
```

Comment:
The number of buckets is inferred as 0 if users did not specify the bucket number when they create a Colocation table, which causes failures in adding new partitions.